apps/api/v2/src/ee/event-types-private-links/services/private-links-input.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from "@nestjs/common";
+
+import {
+  CreatePrivateLinkInput_2024_06_14,
+  UpdatePrivateLinkInput_2024_06_14,
+} from "@calcom/platform-types";
+
+@Injectable()
+export class PrivateLinksInputService {
+  constructor() {}
+
+  transformCreateInput(input: CreatePrivateLinkInput_2024_06_14): CreatePrivateLinkInput_2024_06_14 {
+    return {
+      expiresAt: input.expiresAt,
```

Comment:
What happens if `input.expiresAt` and `input.maxUsageCount` are both undefined since it is possible to submit empty body? Does it do nothing or error?

If it errors and error is not handled either we handle it or add a check here in `transformCreateInput` checking that at least one of them is defined. If it does nothing its fine as is.